### ISSUE TITLE
Ability to change spacing before a comment

### ISSUE THREAD

**Issue by maxdanilov** [looztra/yamkix#11]
Hi there,
Really enjoying your tool, works very well!

One request though would be to be able to set spacing before comments in the output.
Currently, the comments always seem to stay on their previous position, even if actual yaml code has been moved by formatting (because of [ruamel](https://pypi.org/project/ruamel.yaml/) I guess?)

**Example**:
Input file:

```yaml
---
items:
    - part_no:   A4786 # comment
```

Output from yamkix:
```yaml
---
items:
- part_no: A4786       # comment
```

**Feature request:**
I think it should be a tunable parameter, as e.g. [yamllint](https://github.com/adrienverge/yamllint) has a default rule of 2 spaces before a comment.

Example:
```
yamkix --spaces-before-comment 2
```
```yaml
---
items:
- part_no: A4786  # comment
```

**Comment by looztra** [comment]
Thanks for the feedback. I'll look into it.

Le dim. 2 févr. 2020 à 01:04, Max Danilov <notifications@github.com> a
écrit :

**Comment by looztra** [comment]
There doesn't seem to have a trivial fix, I'll have to contact the author of ruamel.yaml to see if something can be done

**Comment by looztra** [comment]
I am working on this issue but I cannot get to the expected result for the moment :

- I can produce an output were all comments will be placed a single space after the content (what the Google CloudCode extension provides for instance) and that will definitely be an option you will be able to use in `yamkix`
- I am still working on the option where the comments will be placed to please yamllint, but I have still things to understand, see <https://stackoverflow.com/questions/60915926/how-to-force-the-comment-position-to-be-valid-with-yamllint-when-load-and-dump-w>.

**Comment by looztra** [comment]
Available in [v0.6.0](https://github.com/looztra/yamkix/releases/tag/v0.6.0)
